content/docs/catalog/troubleshooting-the-catalog/index.md

```diff
@@ -63,6 +62,14 @@ If you end up in such a situation you have an option to remove the individual st
 
 This will unregister the individual entity without affecting other entities or locations in the same discovery tree. The moved/renamed catalog manifest file will be then picked up on the catalog refresh loop.
 
+### My entity still points to the old repository after renaming the repository in GitHub
+
+If you changed the name of your repository, Backstage by default relies on GitHub's redirect approach for renamed repos.
+This means it will not update the repository URL associated with your entity.
+
+You can clean this up by [deleting the entity](/docs/catalog/unregister-components/#manually-ingested-entities) and re-importing it manually using the new URL.
```

Comment:
"re-importing it manually using the new URL."

Is this part necessary? I would have thought that it's better to wait for auto-discovery to pick it up if you're using this? Perhaps that doesn't work?

Reply:
Yea I went with this as I didn't want to go into the complexities of autodiscovery (only admins can check if it will actually match the pattern set, it will only work for GitHub) and it doesn't really make a difference